docker/sp/pom.xml

```diff
@@ -157,6 +157,26 @@
             <groupId>io.cellery.observability</groupId>
             <artifactId>io.cellery.observability.k8s.api.server.client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.oltu.oauth2</groupId>
+            <artifactId>org.apache.oltu.oauth2.client</artifactId>
+            <version>1.0.0</version>
```

Comment:
You can remove the versions from this pom.xml given that it's defined in the root pom.xml